src/servers/typology.server.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+import { sendUnaryData, ServerUnaryCall, UntypedHandleCall } from '@grpc/grpc-js';
+import { CustomerCreditTransferInitiation } from '../classes/iPain001Transaction';
+import { NetworkMap, Typology } from '../classes/network-map';
+import { RuleResult } from '../classes/rule-result';
+import { IFlowFileServiceServer, FlowFileServiceService } from '../models/nifi_grpc_pb';
+import { FlowFileReply, FlowFileRequest } from '../models/nifi_pb';
+import { LoggerService } from '../services/logger.service';
+import { handleTransaction } from '../services/logic.service';
+
+/**
+ * gRPC Health Check
+ * https://github.com/grpc/grpc-node/tree/master/packages/grpc-health-check
+ */
+class Execute implements IFlowFileServiceServer {
+  [method: string]: UntypedHandleCall;
+
+  public async send(call: ServerUnaryCall<FlowFileRequest, FlowFileReply>, callback: sendUnaryData<FlowFileReply>): Promise<void> {
+    const res: FlowFileReply = new FlowFileReply();
+    LoggerService.log('Start - Handle execute request');
+    let networkMap: NetworkMap = new NetworkMap();
+    let ruleResult: RuleResult = new RuleResult();
+    let req: CustomerCreditTransferInitiation = new CustomerCreditTransferInitiation({});
+    try {
+      const reqData = Buffer.from(call.request.getContent_asB64(), 'base64').toString();
+      LoggerService.log(`gRPC string request received with data: ${reqData ?? ''}`);
+      const request = JSON.parse(reqData);
+      networkMap = request.networkMap;
+      ruleResult = request.ruleResult;
+      req = request.transaction;
+    } catch (parseError) {
+      const failMessage = 'Failed to parse execution request.';
+      LoggerService.error(failMessage, parseError, 'ApplicationService');
+      LoggerService.log('End - Handle execute request');
+      res.setResponsecode(0);
+      res.setBody(failMessage);
+      callback(null, res);
+      return;
```

Comment:
Probably we don't need to return anything after 'callback' function? 🤔

Reply:
Code continues even after callback method is called. Callback just basically tells gRPC to respond, but code still continues.